packages/fether-react/src/Send/Send.js

```diff
@@ -12,11 +12,15 @@ import TxForm from './TxForm';
 
 class Send extends Component {
   render () {
+    const {
+      match: { url }
+    } = this.props;
+
     return (
       <Switch>
-        <Route exact path='/send/:tokenAddress' component={TxForm} />
-        <Route path='/send/:tokenAddress/signer' component={Signer} />
-        <Route path='/send/:tokenAddress/sent' component={Sent} />
+        <Route exact path={`${url}/:tokenAddress`} component={TxForm} />
+        <Route path={`${url}/:tokenAddress/signer`} component={Signer} />
```

Comment:
Instead of putting ${url} here, why not put the url with placeholders? `/tokens/:accountAddress/send/:tokenAddress`.

This way in the withRouter HOC, match.params will be filled with the correct params.